src/gcp/cloudscheduler.ts

```diff
@@ -85,14 +86,26 @@ export function updateJob(job: Job): Promise<any> {
  * If one is found, and it is identical to the job parameter, it does nothing.
  * Otherwise, if one is found and it is different from the job param, it updates the job.
  * @param job A job to check for and create, replace, or leave as appropriate.
- * @throws { FirebaseError } if an error response other than 404 is received on the GET call.
+ * @throws { FirebaseError } if an error response other than 404 is received on the GET call
+ * or if cloud resource location is not set.
```

Comment:
This documentation about when it throws does not match what I'm reading in the code below (throws an error on 404).

I'm not sure why the check below was added.

Reply:
There are two calls - it throws on 404 for POST but not 404 for GET. The check below is added because we now want to ensure cloud resource location is there before you deploy scheduled functions. If it's not there, 404 code is returned by GCP and we catch that and print a better message.

Reply:
Changed the doc string to be more clear on this.